nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -1933,7 +1932,17 @@ private Description matchDereference(
       return Description.NO_MATCH;
     }
     if (mayBeNullExpr(state, baseExpression)) {
-      String message = "dereferenced expression " + baseExpression.toString() + " is @Nullable";
+      String message;
+      if (handler.checkIfOptionalGetCall(baseExpression, state)) {
```

Comment:
I think we want a more general mechanism here, not something specific to `Optional.get()`.  We should refactor and let the handler return a different error message if desired given the `baseExpression` and `state`

Reply:
Right. We need an entry point into the base `Handler` interface that allows to override error messages for a given combination of message type (see `MessageTypes`), expression, and state.

Reply:
👍Done